lite/examples/style_transfer/android/app/src/main/java/org/tensorflow/lite/examples/styletransfer/StyleTransferModelExecutor.kt

```diff
@@ -46,17 +46,24 @@ class StyleTransferModelExecutor(
   private var postProcessTime = 0L
 
   init {
-    interpreterPredict = getInterpreter(context, styleTransferPredictModel, useGPU)
-    // The transform model is not optimized for GPU usage yet
-    interpreterTransform = getInterpreter(context, styleTransferTransferModel, useGPU)
+    if (useGPU) {
+      interpreterPredict = getInterpreter(context, styleTransferPredictGPUModel, true)
+      interpreterTransform = getInterpreter(context, styleTransferTransferGPUModel, true)
+    } else {
+      interpreterPredict = getInterpreter(context, styleTransferPredictModel, false)
+      interpreterTransform = getInterpreter(context, styleTransferTransferModel, false)
+    }
   }
 
   companion object {
     private val TAG = "StyleTransferMExec"
     private var styleImageSize = 256
     private var contentImageSize = 384
+    private var bottleneckSize = 100
     private var styleTransferPredictModel = "style_predict_quantized_256.tflite"
     private var styleTransferTransferModel = "style_transfer_quantized_384.tflite"
```

Comment:
These are constants. Please change to val with a capitalized variable name.

Reply:
Done.